dart_test.yaml

```diff
@@ -7,3 +7,42 @@ tags:
     skip: "skipping DDC-specific tests until UIP-2410 is implemented"
   # Tests that should NOT be run in the dev compiler
   no-ddc:
+
+presets:
+  dartdevc:
+    concurrency: 4
+    exclude_tags: no-ddc || dart2js-only
+    platforms:
+      - chrome
+      - vm
+    paths:
+      - test/vm_tests/
+      - test/over_react_component_declaration_test.dart
+      - test/over_react_component_test.dart
+      - test/over_react_dom_test.dart
+      - test/over_react_util_test.dart
+
+  dart2js:
+    include_tags: dart2js-only
```

Comment:
yes, and yes. There's only 2 tests that _need_ to run on dart2js, the rest are running in ddc so are not duplicated here. Now that I'm thinking about it again, it might be better to just run the full suite of tests in dart2js as well. The big consumer of time when running dart2js tests is the build, not the actual tests themselves, so there's very little time cost to run them all.

Reply:
+1 to running the full suite of tests on dart2js